apps/tup-cms/Dockerfile

```
@@ -7,7 +7,7 @@ COPY . /code/
 RUN npx nx build tup-ui
 RUN npx nx build tup-cms-react
 
-FROM taccwma/core-cms:v4.9.0
+FROM taccwma/core-cms:v4.10.0
```

Use v4.10.2 (like #451 does) to get all the conf. e-mail feature updates this PR was waiting for.

```suggestion
FROM taccwma/core-cms:v4.10.2
```